package.json

```diff
@@ -4,7 +4,8 @@
   "description": "TRIM API wrapper",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/mocha/bin/mocha specs.js",
+    "test:watch": "./node_modules/nodemon/bin/nodemon.js --exec \"npm run test\" specs.js"
```

Comment:
And if anyone is interested in using GWT syntax, there are now 2 mocha libs to choose from. The one we use in mappy:
https://github.com/rendro/mocha-given

And a newer more actively maintained version here:
https://github.com/TheLudd/mocha-gwt